### ISSUE TITLE
AutoML Object Detection: Argument 'x' passed to 'unstack' must be numeric tensor, but got string tensor

### ISSUE THREAD

**Issue by ghafran** [tensorflow/tfjs#4437]
With the latest version of AutoML object detection on edge, we are getting this error using the out-of-box object detection model.

```
tfjs:17 Uncaught (in promise) Error: Argument 'x' passed to 'unstack' must be numeric tensor, but got string tensor
    at Qg (tfjs:17)
    at $g (tfjs:17)
    at unstack_ (tfjs:17)
    at unstack__op (tfjs:17)
    at tfjs:17
    at tfjs:17
    at e.t.scopedRun (tfjs:17)
    at e.t.tidy (tfjs:17)
    at lx (tfjs:17)
    at e.<anonymous> (tfjs:17)
```
Model here: [model.zip](https://github.com/tensorflow/tfjs/files/5727267/model.zip)

Below is the html code:
```
<script src="https://unpkg.com/@tensorflow/tfjs"></script>
<script src="https://unpkg.com/@tensorflow/tfjs-automl"></script>
<img id="img" src="test/test.jpg">
<script>
    async function run() {
        const model = await tf.automl.loadObjectDetection('/model/model.json');
        const img = document.getElementById('img');
        const options = { score: 0.5, iou: 0.5, topk: 20 };
        console.log(model.dictionary);
        const predictions = await model.detect(img, options);
        console.log(predictions);
        // Show the resulting object on the page.
        const pre = document.createElement('pre');
        pre.textContent = JSON.stringify(predictions, null, 2);
        document.body.append(pre);
    }
    run();
</script>
```

Library Versions
```
"@tensorflow/tfjs": "^2.8.1",
"@tensorflow/tfjs-automl": "^1.0.0",
```

**Comment by rthadur** [comment]
@tafsiri @lina128

**Comment by lina128** [comment]
Hi @ghafran, I've submitted a PR to fix this. It will be released next week in 2.8.2. For context, the reason this error shows up  in the latest library is that, we started supporting hashtables in models. In this particular model, the hashtable is the label dictionary with string values, so we also need to add string support to underlying ops.

**Comment by ghafran** [comment]
@lina128 Thanks for the update. Looking forward to the fix. Would be good to automate the QA for automl to ensure models work with latest tensorflowjs. Happy to help with that.
